src/mlpack/methods/rann/ra_search_rules.hpp

```diff
@@ -226,6 +226,8 @@ class RASearchRules
                  TreeType& referenceNode,
                  const double oldScore);
 
+  //! Get the minimum number of basecases to be performed.
+  size_t MinBaseCases() const { return k; }
```

Comment:
@rcurtin  I couldn't add this function to all rules class as I don't know what to return for every rules class.

Reply:
That's fair, I'll try to review this soon and when I do I'll provide input for the other rules classes. :+1: